DetectorDescription/DDCMS/plugins/dd4hep/DDDefinitions2Objects.cc

```diff
@@ -860,6 +865,16 @@ void Converter<DDLPosPart>::operator()(xml_h element) const {
   string childName = ns.prepend(ns.attr<string>(e.child(DD_CMU(rChild)), _U(name)));
   Volume parent = ns.volume(parentName, false);
   Volume child = ns.volume(childName, false);
+  xml_dim_t physVolName = e.child(DD_CMU(rPhysVol), false);
```

Comment:
@cvuosalo - why not use <ShapelessSolid> to (re-)store an Assembly?

Reply:
Yes, this tag was my workaround since Assembly had no definition in XML until yesterday. But now that we can define assemblies in XML, this workaround can be dropped and replaced with explicit Assembly definitions, which are much more elegant.